src/public/services/__tests__/AdminSubmissionsService.test.ts

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import MockAxios from 'jest-mock-axios'
+
+import { SubmissionMetadataList } from 'src/types'
+
+import { ADMIN_FORM_ENDPOINT } from '../AdminFormService'
+import {
+  countFormSubmissions,
+  getEncryptedResponse,
+  getFormsMetadata,
+} from '../AdminSubmissionsService'
+
+jest.mock('axios', () => MockAxios)
+
+describe('AdminSubmissionsService', () => {
+  describe('countFormSubmissions', () => {
+    const MOCK_FORM_ID = 'mock–form-id'
+    const MOCK_START_DATE = new Date(2020, 11, 17)
+    const MOCK_END_DATE = new Date(2021, 1, 10)
+
+    it('should call api successfully when all parameters are provided', async () => {
+      // Act
+      const actual = countFormSubmissions({
+        formId: MOCK_FORM_ID,
+        startDate: MOCK_START_DATE,
+        endDate: MOCK_END_DATE,
+      })
+      MockAxios.mockResponse({ data: 123 })
+
+      // Assert
+      await expect(actual).resolves.toEqual(123)
```

Comment:
I would much rather structure this as follows:
```suggestion
      // Arrange
      MockAxios.mockResponse({ data: 123 })
      
      // Act
      const actual = await countFormSubmissions({
        formId: MOCK_FORM_ID,
        startDate: MOCK_START_DATE,
        endDate: MOCK_END_DATE,
      })

      // Assert
      expect(actual).toEqual(123)
```

advantages:
1. more understandable flow. you clearly set up the test then call the tested function, rather than jumping back and forth between set up and function calls.
2. takes advantage of the clarity of `async` `await` syntax. no need to take the extra step of understanding "oh so the Promise is created in one place and awaited on in another place".
3. `.resolves` is quite Jest-specific, whereas `expect(actual).toEqual(123)` is valid syntax for many frameworks. it's a small thing but I have a slight preference for framework-agnostic code, which will be helpful if we migrate test frameworks in 2-3 years.

Reply:
Oh actually I referred to Chin's code for the tests in my $resource PR so would just like to join this discussion. I think the documentation for [mockResponse](https://github.com/knee-cola/jest-mock-axios#axiosmockresponseresponse-requestinfo-silentmode) requires it to be called after the `axios` call. But maybe we use to `MockAxios.get.mockResolvedValueOnce` instead to make the flow clearer

Reply:
MockAxios calls can't be done before (or after) the await... cause it's hijacking the promise lol

Reply:
that's damn stupid

Reply:
we can't actually place it before the call because, as @chowyiyin pointed out, it needs to be after. i think this is a good place to put a comment because it's very specific! i'll try it with  `MockAxios.get.mockResolvedValueOnce` and if that fails, add a comment along with a link to docs explaining this

Reply:
what I don't understand is why elsewhere in the codebase we stick to the arrange/act/assert pattern with near-religious zeal, but suddenly we're happy to make accommodations here for the `MockAxios` library which doesn't even provide any utility above Jest's native mocking? I don't get it, and I would much rather just use `jest.mock` if `MockAxios` is forcing us to contort the structure of our tests

Reply:
Yea, maybe we can switch back to just mocking axios without a third party matcher!

Reply:
yea, i agree that the structure you proposed is better and i'm going to try (and report back) if it works! i think i should've been more mindful of using MockAxios without considering its utility

Reply:
yeap it works! changed imports in this file to use remove mockAxios but idt it's a big enough deal to open a new issue for it right? (or we open but yknow p4)